yorkie/rpc/server.go

```diff
@@ -393,6 +394,7 @@ func (s *Server) WatchDocuments(
 		return err
 	}
 
+	// TODO(dc7303): broadcast
```

Comment:
What does this mean?

Reply:
Sorry. It seems to have been written roughly as a task to be carried out in this PR.
I plan to add an API to notify other Agents of initialization information when WatchDocument is called.